backend/User/validators.py

```diff
@@ -0,0 +1,17 @@
+from django.core.exceptions import ValidationError
+import os
+
+def validate_image(file):
+	valid_mime_types = ['image/jpeg', 'image/png', 'image/gif']
```

Comment:
gifs? really? xD

Reply:
:D I know right, didnt test it yet but on the next PR will :D